sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -260,9 +260,9 @@ void guessLocalWorkSize(int *threadsPerBlock, const size_t *global_work_size,
   (void)minGrid; // Not used, avoid warnings
 
   threadsPerBlock[0] =
-      std::min(static_cast<int>(maxThreadsPerBlock[0]),
-               std::min(static_cast<int>(global_work_size[0]),
-                        static_cast<int>(recommendedBlockSize)));
+      std::min((maxThreadsPerBlock[0]),
+               std::min((global_work_size[0]),
```

Comment:
Just a nitpick, but I would prefer to have the redundant parenthesis removed..

Reply:
Sure! No problem, I added a commit removing them.